rtgui/imagearea.cc

```diff
@@ -170,10 +169,11 @@ CropWindow* ImageArea::getCropWindow (int x, int y)
 
     CropWindow* cw = mainCropWindow;
 
-    for (std::list<CropWindow*>::iterator i = cropWins.begin(); i != cropWins.end(); i++)
-        if ((*i)->isInside (x, y)) {
-            return *i;
+    for (auto cropWin : cropWins) {
```

Comment:
iterator, with templates, and the 2 things that I'm not used to at all in C++, so this syntax is very convenient.

For the `auto` type, it's certainly useful (especially here), but one of the downside is that you don't know the type of your variable by visually inspecting the code. Hopefully Eclipse show us the real type when hovering the variable.